src/egon/data/datasets/etrago_setup.py

```diff
@@ -388,13 +548,17 @@ class EgonPfHvStore(Base):
         source_dict["bgr_inspee"],
         source_dict["bgr_inspeeds"],
         source_dict["bgr_inspeeds_data_bundle"],
-        source_dict["bgr_inspeeds_data_bundle"],
         source_dict["bgr_inspeeds_report"],
+        source_dict["SciGRID_gas"],
     ]
```

Comment:
Are there no sources for DSM, heat and e-mobility storages that should figure here?

Reply:
Yes you're right. At least for DSM there should be a source. @KathiEsterl could you write which one(s) you would cite? I can integrate it to the scripts.

Reply:
Yes, I can: 

We use the concept and parameters by Wilkos work: https://www.sciencedirect.com/science/article/pii/S2666792420300019
Apart from that, there is no work to be cited, I guess.

I am using the following tables to generate my data:
  sources:
    cts_loadcurves:
      schema: 'demand'
      table: 'egon_etrago_electricity_cts'
    ind_osm_loadcurves:
      schema: 'demand'
      table: 'egon_osm_ind_load_curves'
    ind_osm_loadcurves_individual:
      schema: 'demand'
      table: 'egon_osm_ind_load_curves_individual'
    ind_sites_loadcurves:
      schema: 'demand'
      table: 'egon_sites_ind_load_curves'
    ind_sites_loadcurves_individual:
      schema: 'demand'
      table: 'egon_sites_ind_load_curves_individual'
    ind_sites:
      schema: 'demand'
      table: 'egon_industrial_sites'
    ind_sites_schmidt:
      schema: 'demand'
      table: 'egon_schmidt_industrial_sites'
    demandregio_ind_sites:
      schema: 'demand'
      table: 'egon_demandregio_sites_ind_electricity'
      
I am not completely sure in which format you need the sources, just let me know if I should change anything!

Reply:
Okay, actually I need to add all the sources that have been used as an input to all the tables you listed.
But I guess all these tables don't have meta data by now, right?
It's not that easy for me to find out which sources have been used. So I would not add them now but leave a comment that they will be added later.